### ISSUE TITLE
Re-use for inference a trained from scratch model 

### ISSUE THREAD

**Issue by i-mein** [akshaypai/tfClassifier#13]
Hi and thanks for sharing this. I have retrained a new model instance with a new set of images (5 classes). Any advice on how can be used to predict/make inference on my test set images ? i.e. is there a way for the script retrain_model_classifier.py to accept a whole directory of images ?

Also what modifications do we need to make to save the retrained model as .h5 at the end of the training?

Thank you in advance

**Comment by i-mein** [comment]
Actually, I close it as issue since it is more  a question/advice help task. I coppy my post to your blog https://sourcedexter.com/retrain-tensorflow-inception-model/
